src/agent/application.hpp

```diff
@@ -105,7 +107,7 @@ class Application
 
     static void HandleSignal(int aSignal);
 
-    Ncp::ControllerOpenThread &mPlaceHolder;
+    Ncp::ControllerOpenThread mNCP;
```

Comment:
```suggestion
    Ncp::ControllerOpenThread mOpenThread;
```

Reply:
Most references to `ControllerOpenThread` uses `aNcp` or `mNcp`. 

Maybe better to be consistent?

Reply:
No strong preference for both. 

But will keep `mNcp` for now.